userspace/libsinsp/container.cpp

```diff
@@ -705,29 +748,45 @@ sinsp_docker_response sinsp_container_manager::get_docker(const string& api_vers
 
 	json = response;
 
-#endif // CYGWING_AGENT
-
 	if(strncmp(json.c_str(), "HTTP/1.0 200 OK", sizeof("HTTP/1.0 200 OK") -1))
 	{
 		return sinsp_docker_response::RESP_BAD_REQUEST;
 	}
 
+	size_t pos = json.find("{");
+	if(pos == string::npos)
+	{
+		ASSERT(false);
+		return sinsp_docker_response::RESP_ERROR;
+	}
+
+	json = json.substr(pos);
+#endif // CYGWING_AGENT
+
 	return sinsp_docker_response::RESP_OK;
 }
 
 bool sinsp_container_manager::parse_docker(sinsp_container_info* container)
 {
 	string json;
-        sinsp_docker_response resp = get_docker("/v1.24", container->m_id, json);
+#ifndef CYGWING_AGENT
+	sinsp_docker_response resp = get_docker("http://dummy" + m_api_version + "/containers/" + container->m_id + "/json", json);
```

Comment:
You might want to use `localhost` instead of `dummy`, just in case some future version of curl tries to do validation of the hostname part.

Reply:
I agree, will do